newswires/client/src/WireItemList.tsx

```diff
@@ -95,11 +99,31 @@ function decideMainHeadingContent({
 	return 'No headline';
 }
 
-function decideSecondaryCardContent({
+function SecondaryCardContent({
 	headline,
 	subhead,
 	bodyText,
-}: WireData['content']): string | undefined {
+	highlight,
+}: Pick<WireData['content'], 'headline' | 'subhead' | 'bodyText'> & {
```

Comment:
is there a reason to do the `Pick` here, is there harm in just using the type for the whole object... since that's what is being passed in at the call site.

Reply:
Being more explicit about what it was actually using, but I don't feel strongly on this so happy to change 👍